### ISSUE TITLE
[Introspection] `(` is not accepted as character in column name

### ISSUE THREAD

**Issue by janpio** [prisma/prisma2#1081]
Error:
```
error: Unexpected token. Expected one of: field type.
  -->  schema.prisma:2
   | 
 1 | model calculated_feature {
 2 |   aggregate_avg(length) Float?
   |                ^ Unexpected token.
 3 |   id                    Int?
   | 
```
(https://github.com/prisma/introspection-engine-output/pull/5/files#diff-6e67900e5387baadc97d897ddd69110f)

Model:
```

model calculated_feature {
  aggregate_avg(length) Float?
  id                    Int?
  target                String?
}
```
(https://github.com/prisma/introspection-engine-output/pull/5/files#diff-b62a4dcb9989236eecdb6dae00dc4895)

Original DB:
```
CREATE TABLE `calculated_feature` (
	`id` INT(11) NULL DEFAULT NULL,
	`target` VARCHAR(4) NULL DEFAULT NULL COLLATE 'latin1_swedish_ci',
	`aggregate_avg(length)` DECIMAL(13,4) NULL DEFAULT NULL
)
COLLATE='utf8_general_ci'
ENGINE=InnoDB
;

```

![image](https://user-images.githubusercontent.com/183673/70396899-6a632180-1a0d-11ea-91f9-8710685bdd95.png)

**Comment by pantharshit00** [comment]
I can confirm this bug. A temporary workaround here would be to add the @map directive to those fields and changing the identifier to a valid name.

**Comment by janpio** [comment]
fixed via #1139
